scripts/pyinstaller/hooks/hook-dvc.py

```diff
@@ -14,6 +14,8 @@
 datas += copy_metadata("ossfs")
 datas += copy_metadata("sshfs")
 datas += copy_metadata("webdav4")
+datas += copy_metadata("aiohttp")
+datas += copy_metadata("aiohttp_retry")
```

Comment:
There's no way to validate it in CI, we need to merge this and see if this worked or not.